src/themes/default/globals/site.variables

```diff
@@ -9,12 +9,40 @@
 --------------------*/
 
 @fontName          : 'Lato';
+@fontFileName      : 'LatoLatin';
```

Comment:
"Lato v1" has very bad font for Cyrillic. With "LatoLatin", this issue remains.

"System native font" (as defined in Bootstrap v4+) however renders Cyrillic glyphs correctly, can it be defined as a fallback font?

I understand the "LatoLatin" font is smaller in size, but a lot of websites use the compiled dist semantic.css style. Is there any easy way how to use full "Lato" font /wo recompile?

Reply:
> "System native font" (as defined in Bootstrap v4+) however renders Cyrillic glyphs correctly, can it be defined as a fallback font?

Not without recompile by using the systemfont theme for site or changing the `@pageFont` accordingly.

However, you may do a manual overwrite after importing the prebuild semantic.css like this :
```css
*:not(i) {
	font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !important;
}
```
Providing the System fonts as fallback src setting for the font family does not make sense i IMHO, because the Lato font will still be applied/loaded.
I decided against making Systemfonts the default as this will definately break many projects and the main concern was to disable the googlefonts connection.

> Is there any easy way how to use full "Lato" font /wo recompile?

Not without recompile. My intention was to have the same font as before (LatoLatin 2.0 in comparision to 1.0 is at least fixing the missing european diacritic letters which issued by #2355)

But we can of course discuss this 🙂 

@fomantic/maintainers @fomantic/admins 

Should i
- remove the extra Lato Full theme and use the use that as default instead? (Only downside: The font files are 5 times as big as LatoLatin is
- revert the logic: Use Lato Full as default and LatoLatin as optional theme?
- leave the PR as it is

?

Reply:
This is how Google define fonts:

https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400%3B0,700%3B1,400%3B1,700&subset=latin&display=swap

The individual font files are downloaded on the first use, test:

```
<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400%3B0,700%3B1,400%3B1,700&subset=latin&display=swap">

<style>
* { font-family: 'Lato'; font-weight: 400; }
</style>

<p>some text</p>
```

given this behaviour, we can define both LatoLatin and Lato fonts together with `unicode-range` CSS property. This way, full Lato will be supported by default, but downloaded only if needed by the webpage. `unicode-range` property can be also used to define fallback to native system font stack for the glyphs not present even in the full Lato font.

Reply:
I'll check and test that and will adjust the PR accordingly. Thanks for the hint 👍

Reply:
I compared the Lato font with native font stack visually. The native font stack (as defined by Bootstrap) is almost the same as Lato font, only bold font (/w style `font-weight: bold`) is much bolder. `bold` `font-weight` equals to 700 weight. When I changed it to 600, the visual difference is very low. I also checked other popular websites like Github :) and it uses `font-weight` set to 600. So I propose to change `font-weight: bold` in Fomantic-UI to `font-weight: 600` and drop any bundling of Lato font completely. Using native font stack is definitely a trend, the biggest styling framework Bootstrap has choosen it for a reasons. For the user it renders instantly (no download needed) and it covers also the biggest Unicode range consistently (like the `Řeřicha` issue).

Reply:
🤔 Well, at least for windows, i can see a very clear difference using systemfonts vs Lato

|Lato|System|
|-|-|
|![image](https://user-images.githubusercontent.com/18379884/167289357-54c7658c-8b05-4ad8-b7fc-5c63c1968dc4.png)|![image](https://user-images.githubusercontent.com/18379884/167290967-ea3a06af-e04d-44cc-a9b4-5820e825d690.png)|

### Lato
![image](https://user-images.githubusercontent.com/18379884/167290783-df8e94c3-339f-48b7-aa29-4b12c874386e.png)
### System
![image](https://user-images.githubusercontent.com/18379884/167290787-2e59f7da-656e-44e3-8df4-8382a153ba26.png)

Here are two jsfiddles to compare almost all components
Lato https://jsfiddle.net/hpetaxy0/show
System fonts https://jsfiddle.net/lubber/9c3hf7eq/1/show

The minimum immediate different is the bold weight. Yes, if this is changed from `bold` to `600` the thickness is nearly the same as when using Lato. 
However, the font-weight variable `@bold` is unfortunately not used everywhere. font-weight is changed by  individial variables at several places (calendar, button, table...) where this would have to be changed as well everywhere, which, in turn, _could_ have impact on existing projects, even while at the first look, it looks similar/identical. 
Also the line height changes for the systems fonts which in turn also changes the padding which _could_ have undesired impact on existing projects.

So, we will bundle Lato 2.0 with 2.9.0 (using unicode-range as suggested by you) for most possible backward compatibility to existing projects. Other people might have a different opinion about using only system fonts.

Don't get me wrong, i fully understand and mostly agree with your arguments 🙂 

I am currently rewriting the PR to have the full freedom to import as many custom fontfaces with custom font-face properties as desired by a single variable (or none and only stay with native fontstack, which will become the fallback default as suggested by you (instead of an optional theme)).

We can discuss using pure Native fontstack as default for 3.0, which will be as simple as setting `@font: ''` and optional `@importFonts: false` or `@fonts: {}` and the LESS script takes care of that.

Reply:
🤔 Seems i am wrong with my statement about the `@bold` variable, which indeed seems to be used consistenly everywhere. So i will consider changing this from `bold` to 600 in this PR as you suggested (bold/600 look identical using lato and 600 looks nearly equal using systemfont as you explained.

Reply:
🤔 There is one issue when using the systemfonts and bold: The default tag `<b></b>` defaults the font-weight to `bold` which would still show them noticable different when not also set to `600` manually and i believe this is still often used technique to show some inline text as bold.
Yes, we could override this for `strong,b` to 600 in general via `reset.overrides`, but i fear side effects for existing projects as this will also override something outside the `ui` scope.

Reply:
I would not object decision in any direction for now. On one side, we override some global tags like `h1` and set font style globally (on `body`), so overriding `font-weight` of tag `<b>` to `600` outside the `ui` scope is not that problematic.

Nevertheless I slightly prefer to keep `<b>` `font-weight` set to default (`bold`/`700`) as it should be consistent with setting CSS font weight to `bold`/`700`, eg. `<b>test</b>` and `<span style="font-weight: bold;">test</span>` should be strictly equivalent. Also I find the visually thicker bold much more readable in places where "desired for semantic purposes".

Currently, `b` (and `strong`) is actually overridden/set globally to `font-weight: bolder` which does imply font-weight `700` when used with `normal`/`400` font [1]. So I think we do not need any change for `b`/`strong`.

[1] https://css-tricks.com/firefoxs-bolder-default-is-a-problem-for-variable-fonts/ (inconsistent across Chrome/Firefox, but consistent when the base font has `normal`/`400` font weight 😄)

Reply:
I updated the PR and description accordingly. :)